.gitignore

```diff
@@ -4,4 +4,6 @@ main
 .idea
```

Comment:
can you also add

config/*.xml
!config/config.xml
*.iml
.idea
ui/
automated-perf-test
uiServices/test/*.xml